products/green_fast_track/models/product/green_fast_track_bbls.sql

```diff
@@ -49,6 +49,14 @@ flags_wide AS (
             )
             AND variable_type != 'wetlands_checkzones'
         ) AS natural_resource,
+        FILTER (
```

Comment:
creeping on you because I had to see what commit "Maybe?" was doing. Looks like you're missing whatever array_agg comes before this `FILTER`, so sqlfluff is interpreting `FILTER` as a function name rather than a keyword

Reply:
On line 40 I have `bool_or` which is an aggregate function, which `FILTER` on line 41 then modifies

Reply:
And `any` should in fact still be lowercase